rs/moq-native/src/server.rs

```diff
@@ -301,20 +338,20 @@ impl QuicRequest {
 
 #[derive(Debug)]
 struct ServeCerts {
-	certs: Vec<Arc<CertifiedKey>>,
+	certs: RwLock<Vec<Arc<CertifiedKey>>>,
```

Comment:
Why is this a `RwLock` if `ServerReloader` replaces the Arc, instead of modifying it inline?